huaweicloud/services/acceptance/dds/resource_huaweicloud_dds_lts_log_test.go

```diff
@@ -0,0 +1,122 @@
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/dds"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getDdsLtsLogResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	var (
+		getDdsLtsLogHttpUrl = "v3/{project_id}/instances/logs/lts-configs"
+		getDdsLtsLogProduct = "dds"
+	)
+
+	getDdsLtsLogClient, err := cfg.NewServiceClient(getDdsLtsLogProduct, acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating DDS Client: %s", err)
+	}
+
+	getDdsLtsLogPath := getDdsLtsLogClient.Endpoint + getDdsLtsLogHttpUrl
+	getDdsLtsLogPath = strings.ReplaceAll(getDdsLtsLogPath, "{project_id}", getDdsLtsLogClient.ProjectID)
+
+	getDdsLtsLogResp, err := pagination.ListAllItems(
+		getDdsLtsLogClient,
+		"offset",
+		getDdsLtsLogPath,
+		&pagination.QueryOpts{MarkerField: ""})
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving DDS LTS configs: %s", err)
+	}
+
+	getDdsLtsLogRespJson, err := json.Marshal(getDdsLtsLogResp)
+	if err != nil {
+		return nil, fmt.Errorf("error marshaling DDS LTS configs: %s", err)
+	}
+
+	var getDdsLtsLogRespBody interface{}
+	err = json.Unmarshal(getDdsLtsLogRespJson, &getDdsLtsLogRespBody)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling DDS LTS configs: %s", err)
+	}
+
+	instanceLtsConfig := dds.FilterInstanceLtsConfig(state.Primary.ID, getDdsLtsLogRespBody)
+	ltsConfigs := utils.PathSearch("lts_configs", instanceLtsConfig, make([]interface{}, 0)).([]interface{})
+	if len(ltsConfigs) == 0 || !utils.PathSearch("enabled", ltsConfigs[0], false).(bool) {
+		return nil, fmt.Errorf("error retrieving DDS LTS config: the LTS config can not be found")
+	}
+
+	return instanceLtsConfig, nil
+}
+
+func TestAccDdsLtsLog_basic(t *testing.T) {
+	var instance instances.InstanceResponse
+	rName := acceptance.RandomAccResourceName()
+	resourceName := "huaweicloud_dds_lts_log.test"
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&instance,
+		getDdsLtsLogResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDdsLtsLog_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(resourceName, "instance_id", "huaweicloud_dds_instance.instance", "id"),
+					resource.TestCheckResourceAttrPair(resourceName, "lts_group_id", "huaweicloud_lts_group.test", "id"),
+					resource.TestCheckResourceAttrPair(resourceName, "lts_stream_id", "huaweicloud_lts_stream.test", "id"),
+					resource.TestCheckResourceAttr(resourceName, "log_type", "audit_log"),
+					resource.TestCheckResourceAttrSet(resourceName, "enabled"),
```

Comment:
you will wait for the config complete, so it is more better to check whether the value of enabled is true

Reply:
done